### PR TITLE
[15.0][MIG] delivery_free_fee_removal: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 15.0

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT36945

### CODE REVIEW

File: delivery_free_fee_removal/tests/test_delivery_free_fee_removal.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# Copyright 2019 tecnativa - Vicent Cubells
```
Comment (pedrobaeza) [inline delivery_free_fee_removal/tests/test_delivery_free_fee_removal.py]:
```suggestion
# Copyright 2019 Tecnativa - Vicent Cubells
# Copyright 2022 Tecnativa - Víctor Martínez
```

File: delivery_free_fee_removal/tests/test_delivery_free_fee_removal.py [hunk 2]
```diff
@@ -60,6 +63,10 @@ def test_delivery_free_fee_removal_with_fee(self):
                 }
             ],
         )
+        res = self.report_obj._get_report_from_name(
```
Comment (pedrobaeza) [inline delivery_free_fee_removal/tests/test_delivery_free_fee_removal.py:67]: Why do you need to test the report? The same question for the next ones.

Comment (victoralmau) [reply delivery_free_fee_removal/tests/test_delivery_free_fee_removal.py:67]: The purpose of testing the report is: improve the coverage and check that the delivery line is indeed displayed or not (depending on the use case).

Comment (pedrobaeza) [reply delivery_free_fee_removal/tests/test_delivery_free_fee_removal.py:67]: OK, I see now that there's a modification on the sales report.

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Functional and code review :+1: Thanks!

**Pull request merged**